### DISCUSSION TITLE
How to hide 1st option in dropdown using tailwind CSS

### DISCUSSION THREAD

**Discussion by MinalPatilRPA** [tailwindlabs/tailwindcss#12186] [Help]
Hi Team,

How to hide 1st option in dropdown using Tailwind CSS.

My Code:-

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/3366e7c1-9ef7-4901-8d58-1dd94750dcc7)

My output:- 

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/1b7d2bc3-997d-457f-b208-8e6393806b59)

But the expected output is:- 

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/9b65f82b-61a5-46c7-943b-46a0e63b0c2d)

**Comment by wongjn** [comment]
I'm assuming that you have the expected and actual output screenshots mistakenly reversed. You could consider applying `display: none` or the `hidden` class to the `<option>` element you'd like hidden: https://play.tailwindcss.com/BEgshdyTYQ

**Comment by MinalPatilRPA** [comment]
![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/4640314d-642d-4195-8ba9-afbbbaa9e713)

I expect output like as shown in the above fig.


My code:-

               <div class=" flex relative items-center">
                            
                            <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" class="text-white absolute top-[10px] mx-3" data-name="Languages" aria-hidden="true"><path fill-rule="evenodd" clip-rule="evenodd" d="M10.7668 5.33333L10.5038 5.99715L9.33974 8.9355L8.76866 10.377L7.33333 14H9.10751L9.83505 12.0326H13.4217L14.162 14H16L12.5665 5.33333H10.8278H10.7668ZM10.6186 9.93479L10.3839 10.5632H11.1036H12.8856L11.6348 7.2136L10.6186 9.93479ZM9.52722 4.84224C9.55393 4.77481 9.58574 4.71045 9.62211 4.64954H6.41909V2H4.926V4.64954H0.540802V5.99715H4.31466C3.35062 7.79015 1.75173 9.51463 0 10.4283C0.329184 10.7138 0.811203 11.2391 1.04633 11.5931C2.55118 10.6795 3.90318 9.22912 4.926 7.57316V12.6667H6.41909V7.51606C6.81951 8.15256 7.26748 8.76169 7.7521 9.32292L8.31996 7.88955C7.80191 7.29052 7.34631 6.64699 6.9834 5.99715H9.06968L9.52722 4.84224Z" fill="currentColor"></path></svg> 

                            
                            <select name="" id="" class="bg-black text-white border border-gray-500  rounded sm:p-[6px_36px]  appearance-none sm:h-[37px] ">
                              <option value=""  >English</option>
                              <option value="">हिंदी</option>
                           </select>  

                           
                           <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" class="text-white absolute right-3" data-name="CaretDown" aria-hidden="true"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.5976 6.5C11.7461 6.5 11.8204 6.67956 11.7154 6.78457L8.23574 10.2643C8.10555 10.3945 7.89445 10.3945 7.76425 10.2643L4.28457 6.78457C4.17956 6.67956 4.25393 6.5 4.40244 6.5H11.5976Z" fill="currentColor"></path></svg>                       
            </div>


The output of my code is:-

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/47512524-a451-4d08-b8a2-8afe437828e2)



**Note:- I am trying to design a responsive drop-down menu as per the screen. 1st dig. shows the screen is a mobile screen and 2nd dig. shows the screen is a lapi screen.
1. screen size 375 it will show like 1st dig.
2. screen size is above 375(376-1024) it will show like 2nd dig.
**

**Reply by wongjn** [reply]
Consider checking my earlier comment for a possible solution: https://github.com/tailwindlabs/tailwindcss/discussions/12186#discussioncomment-7263345

**Reply by MinalPatilRPA** [reply]
its not working

**Reply by wongjn** [reply]
The solutions seem to work in Tailwind Play (https://play.tailwindcss.com/x1lhhbPBZG), which could mean some misconfiguration or conflict in your project.

For hiding the selected option on smaller viewports, you could make the text color transparent: https://play.tailwindcss.com/KWrTX8miHO?size=520x720

**Comment by iazaran** [comment]
If we need to display something as placeholder at top, we need add it somehow in the options like:
```html
        <option value="" disabled selected>Select a language</option>
        <option value="option1">HTML</option>
        <option value="option2">JavaScript</option>
        <option value="option3">Tailwind CSS</option>
```

**Reply by MinalPatilRPA** [reply]
I want to make a responsive website. 

screen size is Min-640 then output is :- 


![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/c2c7722a-6c17-4dc0-bdf6-5e592092e460)


screen size is min-375 then output is:- 

 
![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/46464b1e-f01e-4acb-b3ee-b48bdc8ab5d4)


MyCode:-

          <div class=" flex relative items-center">
                        
                        <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" class="text-white absolute top-[10px] mx-3" data-name="Languages" aria-hidden="true"><path fill-rule="evenodd" clip-rule="evenodd" d="M10.7668 5.33333L10.5038 5.99715L9.33974 8.9355L8.76866 10.377L7.33333 14H9.10751L9.83505 12.0326H13.4217L14.162 14H16L12.5665 5.33333H10.8278H10.7668ZM10.6186 9.93479L10.3839 10.5632H11.1036H12.8856L11.6348 7.2136L10.6186 9.93479ZM9.52722 4.84224C9.55393 4.77481 9.58574 4.71045 9.62211 4.64954H6.41909V2H4.926V4.64954H0.540802V5.99715H4.31466C3.35062 7.79015 1.75173 9.51463 0 10.4283C0.329184 10.7138 0.811203 11.2391 1.04633 11.5931C2.55118 10.6795 3.90318 9.22912 4.926 7.57316V12.6667H6.41909V7.51606C6.81951 8.15256 7.26748 8.76169 7.7521 9.32292L8.31996 7.88955C7.80191 7.29052 7.34631 6.64699 6.9834 5.99715H9.06968L9.52722 4.84224Z" fill="currentColor"></path></svg> 

                        
                        <select name="" id="" class="bg-black text-white border border-gray-500  rounded sm:p-[6px_36px]  appearance-none sm:h-[37px] ">
                          <option value=""  >English</option>
                          <option value="">हिंदी</option>
                       </select>  

                       
                       <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" class="text-white absolute right-3" data-name="CaretDown" aria-hidden="true"><path fill-rule="evenodd" clip-rule="evenodd" d="M11.5976 6.5C11.7461 6.5 11.8204 6.67956 11.7154 6.78457L8.23574 10.2643C8.10555 10.3945 7.89445 10.3945 7.76425 10.2643L4.28457 6.78457C4.17956 6.67956 4.25393 6.5 4.40244 6.5H11.5976Z" fill="currentColor"></path></svg>                       
           </div>

**Reply by iazaran** [reply]
This is different issue based on the title. We can cover your select by 2 divs like
```html
<div class="grid grid-cols-12 gap-4">
    <div class="col-span-3 bg-gray-100 px-2 py-4">
```
Now we can use the `flex w-full` for your class of `select`
And `block w-full` for each `option`
So it will be responsive, but will be affected by size. So we need to use different `w-...` for each size breakdowns.
